### ISSUE TITLE
Duda sobre Debug.Log

### ISSUE THREAD

**Issue by antu221** [glantucan/puzzle_game#7]
Para el ejercicio 24 quiero que en en la consola me muestre cada vez la velocidad y la distancia al comenzar para comprobar que es realmente aleatorio a cada vez que lo ejecuto.

He desmarcado el "Clear on play" para poder comparar los resultados y el problema surge cuando escribo una tercera linea para separar cada resultado en el debug. 

Este es el codigo que uso:

``` C#
    Debug.Log ("v =" + v);
    Debug.Log ("controlPoint =" + controlPoint);
    Debug.Log ("-----------------");
```

La tercera linea solo sale en consola la primera vez que ejecuto el juego:
![captureb](https://cloud.githubusercontent.com/assets/23030041/19761806/acf5fca4-9c37-11e6-919f-4f3caf35d7e4.PNG)

No es de vital importancia, pero no entiendo porque solo la escribe una vez.

(He intentado ponerle una etiqueta como "Duda" para esta entrada pero no he encontrado donde ponerlo.)

**Comment by JorgeHurtado** [comment]
Esto te pasa porque tienes el Collapse activado. Al tener esto marcado, todos los errores, avisos o Debugs que se repiten en la consola te los junta en una sola línea. 

Si te fijas a la derecha de la imagen que adjuntas, verás que te marca cuantas veces se repite cada línea. En caso de la línea de guiones sale un 4 a la derecha marcando que se repite 4 veces en tu código (en lo que llevas de ejecución).

Prueba a desmarcarlo, yo creo que es eso lo que preguntabas.

Con respecto a lo de las etiquetas, creo que sólo puede ponerlas glantucan, que es el propietario del repositorio dónde está puzzle_game.

**Comment by glantucan** [comment]
Correcto, gracias por contestar a tu compañero, Jorge. A mi se me había pasado. :)
